langlib/langlib-test/src/test/resources/test-src/regexp_test.bal

```diff
@@ -624,12 +638,34 @@ function testReplaceAll() {
     assertEquality("151311", result51);
     string result52 = regExpr4.replaceAll(str5, replacementFunctionForReplaceAll, 6);
     assertEquality("1000001311", result52);
+
+    var regExpr5 = re `\p{S}`;
+    string str6 = "🔜 #RayoBarça 🔵🔴 https://t.co/iHDUx7EmFJ";
+    string result61 = regExpr5.replaceAll(str6, "");
+    assertEquality(" #RayoBarça  https://t.co/iHDUx7EmFJ", result61);
+    string result62 = regExpr5.replaceAll(str6, "", 12);
+    assertEquality("🔜 #RayoBarça  https://t.co/iHDUx7EmFJ", result62);
+
+    var regExpr6 = re `(\d{2})/(\d{2})/(\d{4})`;
+    string str7 = "04/05/2023, 05/05/2023";
+    string result71 = regExpr6.replaceAll(str7, updateDateFormat);
+    assertEquality("2023-05-04, 2023-05-05", result71);
```

Comment:
Shall we add a test for replace functions with `\p{sc=COMMON}` unicode property escape.